roles/etcd/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Install etcd
+  yum: pkg=etcd state=installed disable_gpg_check=yes
+
+- name: Write etcd global config file
+  template: src=etcd.conf.j2 dest=/etc/etcd/etcd.conf
```

Comment:
It seems like most other occurences of template follow Jhon's suggestion, I'll change it.